src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -160,15 +161,17 @@ export async function launchGame(
     // Native
     if (isNative) {
       logInfo(
-        `launching native sideloaded game: ${executable} ${launcherArgs ?? ''}`,
+        `launching native sideloaded game: ${executable} ${extraArgs.join(
+          ' '
+        )}`,
```

Comment:
Prettier sure does destroy the formatting here
Creating a separate `const extraArgsJoined = extraArgs.join(' ')` above and using it here might be "cleaner"